ld/armv7m4-stm32l4x6.ldt

```diff
@@ -0,0 +1,51 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Operating system loader
+ *
+ * Linker Template and Platform Config for STM32L4
+ *
+ * Copyright 2021-2022 Phoenix Systems
+ * Author: Gerard Swiderski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+
+#ifndef ARMV7M4_STM32L4X6_LDT
+#define ARMV7M4_STM32L4X6_LDT
+
+
+/* Platform specific definitions */
+#define SIZE_PAGE  0x200
+#define SIZE_STACK (8 * SIZE_PAGE)
+#define SIZE_HEAP  (8 * SIZE_PAGE)
+
+
+#if defined(__LINKER__)
+
+/* Memory map setup */
+MEMORY
+{
+	m_sram2   (rwx) : ORIGIN = 0x20040000, LENGTH = 256k
+	m_sram1   (rwx) : ORIGIN = 0x20010000, LENGTH = 192k
+	m_flash2  (rx)  : ORIGIN = 0x08000000, LENGTH = 512k
+	m_flash1  (rx)  : ORIGIN = 0x00000000, LENGTH = 512k
+}
+
+/* FLASH image */
+REGION_ALIAS("PLO_IMAGE", m_flash2);
+REGION_ALIAS("TCM_TEXT", m_flash2);
```

Comment:
Everything resides inside of flash1 - flash2 is a second banks, reserved for OTA purpose

Reply:
Thanks, changed to `flash1`